.github/workflows/continuous-build.yml

```diff
@@ -43,6 +43,12 @@ jobs:
         run: make packaging
         env:
           VERSION: ${{ github.sha }}
+
+      - name: Build legacy archives
+        if: ${{ runner.os == 'Linux' }}
+        run: make build-package-legacy
```

Comment:
And as long as we're ok running on continuous build too (I think so as commented), I think it makes sense to wire up the dependency in the makefile instead of here. Alternatively, let's add the step to the release build workflow in this PR so it's easy to compare

Reply:
Wired up the `build-package-legacy` job to `packaging` in makefile. Now it will run on contunuous build and release build as well.